extension/entitystore/extension.go

```diff
@@ -36,6 +36,8 @@ const (
 	Type                 = "Type"
 	Name                 = "Name"
 	Environment          = "Environment"
+	successfulRefresh    = time.Hour * 24
+	failureRefresh       = time.Minute * 5
```

Comment:
I'm definitely not married to these intervals, particularly the failure one. Not sure what a good interval is for this, but one request per 5 minutes should still be much better than X amount of PLE calls.